js/tests/unit/tab.js

```diff
@@ -92,6 +92,24 @@ $(function () {
     assert.strictEqual($('#qunit-fixture').find('.active').attr('id'), 'home')
   })
 
+  QUnit.test('should activate element by tab id in nav list', function (assert) {
+    assert.expect(2)
+    var tabsHTML = [
+      '<nav class="nav">',
+      '<div><a href="#home">Home</a></div>',
+      '<div><a href="#profile">Profile</a></div>',
```

Comment:
Should these lines still have `<div>`s, or can we safely remove them since we removed them from the visual example?